hooks/terraform_docs.sh

```diff
@@ -159,13 +159,24 @@ function terraform_docs {
   if [[ "$args" != *"--config"* ]]; then
     local tf_docs_formatter="md"
 
-  # Suppress terraform_docs color
   else
 
     local config_file=${args#*--config}
     config_file=${config_file#*=}
     config_file=${config_file% *}
 
+    # Prioritize `.terraform-docs.yml` `output.file` over
+    # `--hook-config=--path-to-file=` if it set
+    local output_file
+    # Get latest non-commented `output.file` from `.terraform-docs.yml`
+    output_file=$(grep -A1000 -e '^output:$' "$config_file" | grep ' file:' | grep -v '#' | tail -n 1 || true)
+
+    if [ "$output_file" ]; then
```

Comment:
Bashism for consistency with other conditionals
```suggestion
    if [[ $output_file ]]; then
```